src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -1,32 +1,32 @@
 <script lang="ts">
+import path from "path";
```

Comment:
I very much appreciate the thought of ordering imports here in the "old" file too! Ideally it would be done in a separate commit since now it's kinda hard to say what changes are actually due to moving the helper function what are just reordering. Also the ordering here seems to be "import X" instead of "from X"? Not a blocker.